securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -502,10 +502,14 @@ Options are same as for the createLoginUrl method but 'action' is set to 'regist
 
 Redirects to the Account Management Console.
 
-====== createAccountUrl()
+====== createAccountUrl(options)
 
 Returns the URL to the Account Management Console.
 
+Options is an Object, where:
+
+* redirectUri - Specifies the uri to redirect to when redirecting back to the application.
```

Comment:
Make uri all capitals: URI

Reply:
The document is not consistent here:

lowercase uri:
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L429
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L85
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L451
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L487

uppercase uri:
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L11
* https://github.com/keycloak/keycloak-documentation/blame/8b825608ad7fb866a1e161553ecdd287f4462a59/securing_apps/topics/oidc/javascript-adapter.adoc#L16

The lower case variant sticks to all other uris in that section.

Reply:
Good catch @andymunro, there is an inconsistency in the code sadly.
Yet the lettercase of the documented option needs to respect the letter case in the respective code PR, so it is correct here.